paris-annotations/src/main/java/com/airbnb/paris/annotations/ParisConfig.java

```diff
@@ -12,4 +12,6 @@
     String defaultStyleNameFormat() default "";
 
     Class<?> rClass() default Void.class;
+
+    boolean namespacedResourcesEnabled() default false;
```

Comment:
I will use this in https://github.com/BenSchwab/paris/pull/1 instead of validateDefaultStyle as if you use namespacedResources you will always want that on.